orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -169,7 +170,7 @@ class RunTaskHandler(
             } else if (e is TimeoutException && stage.context["markSuccessfulOnTimeout"] == true) {
               queue.push(CompleteTask(message, SUCCEEDED))
             } else {
-              if (e !is TimeoutException) {
+              if (e !is TimeoutException && e !is UserException) {
```

Comment:
oh, this is actually from a related but separate change. but might as well go together. Basically, we shouldn't log errors for exceptiosn that are not in "our" control. E.g. bad SpEL expression and any other `UserExceptions`